minasan/minasan.go

```diff
@@ -23,14 +23,14 @@ type Minasan struct {
 }
 
 // Targets return mails, group, project from a mail name
-func (m *Minasan) Targets(mailName string) ([]string, string, string, error) {
+func (m *Minasan) Targets(mailName, lastChanceMail string) ([]string, string, string, error) {
```

Comment:
`lastChanceMail` is a setting, not an argument. It's belong to `Minasan` struct.